app/components/listing/ListingInfoTable.tsx

```diff
@@ -7,7 +7,7 @@ interface ListingInfoTableProps<T = any> {
   children?: ReactNode;
 }
 
-export const ListingInfoTable = <T extends unknown>({
+export const ListingInfoTable = <T,>({
```

Comment:
Based on the underlying React FC type signature this is the way to add a generic that works without any extra ceremony.